tcms/xmlrpc/tests/test_serializer.py

```diff
@@ -123,66 +127,103 @@ def setUpTestData(cls):
         TestCaseFactory(author=cls.case_author, default_tester=None, plan=[cls.plans[2]])
         TestCaseFactory(author=cls.case_author, default_tester=None, plan=[cls.plans[2]])
         TestCaseFactory(author=cls.case_author, default_tester=None, plan=[cls.plans[2]])
-        cls.plans = TestPlan.objects.filter(pk__in=[plan.pk for plan in cls.plans])
+
+        plan_keys = []
+
+        for plan in cls.plans:
+            plan_keys.append(plan.pk)
+
+        cls.plans = TestPlan.objects.filter(pk__in=plan_keys)
         cls.plan_serializer = MockTestPlanSerializer(TestPlan, cls.plans)
 
         cls.cases = [
             TestCaseFactory(author=cls.case_author, default_tester=None),
             TestCaseFactory(author=cls.case_author, default_tester=None),
             TestCaseFactory(author=cls.case_author, default_tester=None),
         ]
-        cls.cases = TestCase.objects.filter(pk__in=[case.pk for case in cls.cases])
+
+        case_keys = []
+
+        for case in cls.cases:
+            case_keys.append(case.pk)
+
+        cls.cases = TestCase.objects.filter(pk__in=case_keys)
         cls.case_serializer = MockTestCaseSerializer(TestCase, cls.cases)
 
         cls.products = [ProductFactory(), ProductFactory(), ProductFactory()]
-        cls.products = Product.objects.filter(pk__in=[product.pk for product in cls.products])
+
+        product_keys = []
+
+        for product in cls.products:
+            product_keys.append(product.pk)
+
+        cls.products = Product.objects.filter(pk__in=product_keys)
         cls.product_serializer = MockProductSerializer(Product, cls.products)
 
     def test_get_values_fields_mapping(self):
+        # pylint: disable=protected-access
```

Comment:
putting this comment here will disable the warning for the whole file from this point on. If this is the point put it on the first line of the file, and delete the other comments, if not - put it on the line of the statement you want to disable the warning for.

Personally, I don't see a problem to disable this warning for this file as it is a test. 

Since tests follow their own rules and we don't need most of the `pylint` checks in them anyway, we can considering extracting this rules, and putting them in a `pylintrc` file, where we can define which checks we want or don't want for files that match a regex (e.g. `*test` ) and not have to write these `#disable` comments in every test file. @atodorov what do you think about that?